apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx

```diff
@@ -195,6 +194,41 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
                         </Hint>
                     </Grid.Column>
                 </Grid.Row>
+                <Grid.Row columns={ 1 }>
+                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                        <Field
+                            name="replyToLogout"
+                            label={ 
+                                t("console:develop.features.applications.forms.inboundSTS.fields.replyToLogout.label") 
+                            }
+                            required={ true }
+                            requiredErrorMessage={
+                                t("console:develop.features.applications.forms.inboundSTS.fields.replyToLogout" +
+                                    ".validations.empty")
+                            }
+                            placeholder={
+                                t("console:develop.features.applications.forms.inboundSTS.fields.replyToLogout" +
+                                    ".placeholder")
+                            }
+                            validation={ (value: string, validation: Validation) => {
+                                if (!FormValidation.url(value)) {
+                                    validation.isValid = false;
+                                    validation.errorMessages.push(
+                                        t("console:develop.features.applications.forms.inboundSTS.fields." +
+                                            "replyToLogout.validations.invalid")
+                                    );
+                                }
+                            } }
+                            type="text"
+                            value={ initialValues?.replyToLogout }
+                            readOnly={ readOnly }
+                            data-testid={ `${ testId }-reply-to-logout-url-input` }
```

Comment:
Let's use `data-componentid` if possible.

Reply:
data-testid is used in other places in this page. Is it okay to switch only in this place?

Reply:
+1

Reply:
Updated https://github.com/wso2/identity-apps/pull/4518/commits/51b130fc62dde0229feff37fe7bd25dac7954c59